self-core-impl/src/main/java/com/selfxdsd/core/BitbucketCommits.java

```diff
@@ -127,27 +124,17 @@ public Commit getCommit(final String ref) {
 
     @Override
     public Commit latest() {
-        final Resource resource = this.resources.get(this.commitsUri);
+        final Resource resource = this.resources
+            .get(URI.create(this.commitsUri + "?pagelen=1"));
```

Comment:
@amihaiemil added small optimizaiton here, default pagelen is 30, but since we need only 
the latest one, pagelen 1 is enough.